### ISSUE TITLE
signup page and login page creation 

### ISSUE THREAD

**Issue by Subhams-GIT** [braydenidzenga/Brogram#24]

**Comment by kolapowariz** [comment]
I would be happy to take on this issue. Could you please assign it to me?

**Comment by Anirban780** [comment]
@Subhams-GIT if I may, can I work on Login Page only ?

**Comment by Subhams-GIT** [comment]
> @Subhams-GIT if I may, can I work on Login Page only ?
let the owner decide if he can assign issues to both of us

**Comment by Anirban780** [comment]
> > @Subhams-GIT if I may, can I work on Login Page only ?
> > let the owner decide if he can assign issues to both of us

Yes, I do not mind at all. It is also better if work is divided among us.

**Comment by braydenidzenga** [comment]
@Subhams-GIT @Anirban780 I have assigned this issue over to both of you.

**Comment by Unnati2603** [comment]
If this is not yet implemented or stale, can I be assigned this issue?
@braydenidzenga
